### ISSUE TITLE
Cannot find namespace angular

### ISSUE THREAD

**Issue by vasuneet** [angular/angular.io#3267]
i used the following statement in typescript source file as below referred from angular.io documentation
declare var angular:angular.IAngularStatic;

But it giving me error -
Cannot find namespace angular.

I tested the same in quickstart project also. same issue

I am referring this from the following url -

https://angular.io/docs/ts/latest/guide/upgrade.html#!#making-angularjs-dependencies-injectable-to-angular

**Comment by Foxandxss** [comment]
Are you installing Angular types?

**Comment by vasuneet** [comment]
yes. However in compiling no issue but in IDE it is showing this.
